Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM python:2.7
+
+COPY . /securitybot
+
+env PYTHONPATH $PYTHONPATH:/securitybot
+
+RUN apt-get update && \
+    apt-get install -y mysql-client && \
```

Comment:
I'm not installing MySQL server in the container.  Just installing client and libs as it's a prereq for the python MySQLDB lib.  I'm also using the client to test for database connection and presence before starting the bot.  I could probably change these test to a Python script rather than call the client directly, should seemed to be less trouble this way.

For deployment, i'm working on a docker-compose with MySQL and a Kubernetes deployment manifest in a separate repo.  Can add them here when they're ready if you'd like,